vtest/gen.bat

```diff
@@ -104,8 +104,8 @@ echo     ^<link rel="stylesheet" type="text/css" href="style.css"^> >> %F%
 echo   ^<head^> >> %F%
 echo   ^<body^> >> %F%
 echo     ^<div id="topbar"^> >> %F%
-echo       ^<span^>Current^</span^> >> %F%
-echo       ^<span^>Reference^</span^> >> %F%
+echo       ^<span^>After PR^</span^> >> %F%
+echo       ^<span^>Before PR^</span^> >> %F%
```

Comment:
Shouldn't these be updated also in the files "gen" and "gen-compare"?

Reply:
oh right, forgot that.

Reply:
done